client/forgotPassword/forgotPassword.ctrl.js

```diff
@@ -6,7 +6,7 @@ module.exports = class ForgotPassword {
     this.$mdToast = $mdToast;
     this.errorHandler = errorHandler;
   }
-  sendRequest () {
+  sendPasswordRequest () {
```

Comment:
Nit: This isn't really requesting a password (it's requesting a password reset), but it's an internal function so whatever.